cmd/network/evpn-vrf.go

```diff
@@ -27,14 +27,17 @@ func CreateVRF() *cobra.Command {
 		Use:   "create-vrf",
 		Short: "Create a VRF",
 		Run: func(_ *cobra.Command, _ []string) {
+			var vniparam *uint32
 			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 			evpnClient, err := network.NewVRF(addr)
 			if err != nil {
 				log.Fatalf("could not create gRPC client: %v", err)
 			}
 			defer cancel()
-
-			vrf, err := evpnClient.CreateVrf(ctx, name, vni, loopback, vtep)
+			if vni != 0 {
```

Comment:
can you use `MarkFlagsMutuallyExclusive` or `MarkFlagsRequiredTogether` ?

Reply:
I believe its not possible as LoopbackIpPrefix is only mandatory flag, vni and vtep are optional.